xmodule/modulestore/inheritance.py

```diff
@@ -238,6 +238,17 @@ class InheritanceMixin(XBlockMixin):
         scope=Scope.settings
     )
 
+    optional_content = Boolean(
```

Comment:
Maybe something similar to `optional_completion` would be better? The current name feels like this is related to grading, not the completion.

Same with the `display_name`, the name of the new Studio editor, etc.

Reply:
good idea. will rename it.